tools/generate-classes/generate.js

```diff
@@ -315,35 +315,47 @@ function generate(options, schema, writers) {
   );
 
   const jsonHandlerHeadersImpl = lodash.uniq([
+    NameFormatters.getJsonHandlerIncludeFromName(base, readerNamespace),
     ...lodash.flatten(properties.map((property) => property.readerHeadersImpl)),
   ]);
+
+  if (thisConfig.extensionName) {
+    jsonHandlerHeadersImpl.push("<CesiumUtility/ExtensibleObject.h>");
+    jsonHandlerHeadersImpl.push("<any>");
```

Comment:
This shouldn't be needed, because ExtensibleObject.h already includes it.

Reply:
This was likely added because `misc-include-cleaner` doesn't allow indirect dependencies.

Reply:
Oh right, of course, sorry for the noise!